internal/compiler-bridge/src/main/java/xsbti/ConsoleFactory.java

```diff
@@ -0,0 +1,9 @@
+package xsbti;
+
+// Ported over from sbt-pamflet.
```

Comment:
Small suggestion: Can we remove these comments? Not useful anymore, sbt-pamflet will probably go away in next sbt 1.0 :smile:.